extensions/replication/ReplicationAPI.js

```diff
@@ -67,23 +69,31 @@ class ReplicationAPI {
      */
     static sendDataMoverAction(producer, action, log, cb) {
         const { bucket, key } = action.getAttribute('target');
-        const kafkaEntries = [{ key: `${bucket}/${key}`,
-                                message: action.toKafkaMessage() }];
-        producer.sendToTopic(dataMoverTopic, kafkaEntries, (err, reports) => {
+        const toLocation = action.getAttribute('toLocation');
+        const locationConfig = locations[toLocation];
+        let topic = dataMoverTopic;
+        if (locationConfig && locationConfig.isCold) {
+            topic = `${constants.coldStorageArchiveTopicPrefix}-${toLocation}`;
+        }
+        const kafkaEntries = [{
```

Comment:
the `archive` request message should have a different format.
looking at https://github.com/scality/sorbet/blob/a017505e8e52f18403d24af20ef86761b64c0c72/forwarder/kafka_forwarder.go#L46


it should be something like: 
```
key: 
value: {
  bucketName,
  objectKey,
  objectVersion,
  size,
  requestId,
}
```

@RachedBenMustapha can you please confirm this

Reply:
Confirmed. And to be future proof (I'm currently working on AssumeRole during the processing of these messages) we'll need the `accountId` as well.